fix: move ActionRow outside the ReflexContainer

ojeytonwilliams opened the pull request:
It does not need to be resizable.

There's a lot of diff noise, but the only change is that the ActionRow has been pulled out of the ReflexContainer.

ojeytonwilliams committed:
fix: move ActionRow outside the ReflexContainer

It does not need to be resizable.

Sembauke commented:
The bottom of the editor seems to get cut off for me 
![image](https://user-images.githubusercontent.com/46919888/134345194-5c7c20a2-c29b-47f2-9264-c47f2bd1f914.png)

ojeytonwilliams committed:
fix: make DesktopLayout aware of ActionRow

The ReflexContainer seems to need to be inside a display: flex component
to correctly compute the sizes of it's children.

ojeytonwilliams committed:
refactor: remove unused props

ojeytonwilliams commented:
Good catch, Sem.  Looks like `ReflexContainer`s expect to be children of flex containers.  Not sure why, but this should fix it.
